File: src/test/java/test/GetOrganizationTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class GetOrganizationTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        GetOrganizationDto request = new GetOrganizationDto();
        request.setOrganizationCode("organizationCode_7989");
        request.setWithCustomData(Boolean.TRUE);

        OrganizationSingleRespDto response = client.getOrganization(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/GetOrganizationTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.GetOrganizationDto{
        OrganizationCode: "organizationCode_2466",
        WithCustomData: false,
    }
    response := client.GetOrganization(&request)
    fmt.Println(response)
}

